src/WebJobs.Script.WebHost/Middleware/HttpExceptionMiddleware.cs

```diff
@@ -1,33 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Features;
-
-namespace Microsoft.Azure.WebJobs.Script.WebHost.Middleware
-{
-    internal class HttpExceptionMiddleware
-    {
-        private readonly RequestDelegate next;
-
-        public HttpExceptionMiddleware(RequestDelegate next)
-        {
-            this.next = next;
-        }
-
-        public async Task Invoke(HttpContext context)
-        {
-            try
-            {
-                await this.next.Invoke(context);
-            }
-            catch (HttpException httpException)
-            {
-                context.Response.StatusCode = httpException.StatusCode;
-                var responseFeature = context.Features.Get<IHttpResponseFeature>();
-                responseFeature.ReasonPhrase = httpException.Message;
```

Comment:
No longer setting ReasonPhrase. Apparently it's not even supported in HTTP 2.0 anyways